### DISCUSSION TITLE
Make website sidebar scrollable independently from content

### DISCUSSION THREAD

**Discussion by Aloe-recite** [community-scripts/ProxmoxVE#113] [Ideas]
Hi, when browsing the website, if you scroll the page down and open one of the last categories or one of the longer ones, you then need to scroll up again to the see top of the content area. It would be better to have the sidebar and main content scroll independently, to avoid having to do all this up/down movement, especially evident if you are on a small laptop or a tablet and are opening one after the other subsequent entries.
See the attached screenshot, I just clicked on Zabbix but have to scroll up to see the content. If I then want to click on WatchYourLan or Uptime Kuma, I have to scroll down again, click, and up again to read, and so forth.
Just a minor thing of course, but flet like suggesting it anyway.

![Screenshot](https://github.com/user-attachments/assets/c8de42c0-800f-473c-a1b9-321ccad42a2f)

**Comment by MickLesk** [comment]
@BramSuurdje

**Comment by bramsuurdje** [comment]
PR has been submitted #120
